dist/ledger/README-cli.md

```diff
@@ -40,12 +40,11 @@ Press [Enter] to continue
 
 Pressing `Enter` will proceed with the installation. Pay attention to the instructions both on the terminal and on the device's screen, since a few confirmations are required.
 ```
-Removing the Bitcoin App...
+Removing the existing installed apps (if any)...
 The Ledger will prompt for 'Allow Unknown Manager'. Please accept it.
+Removing the UX App...
```

Comment:
Is this still in the actual output?

Reply:
Yes, we do `echo -e "\e[1;32mRemoving the $app App...\e[0m"` in `setup` script, this was copied directly from the script output.

Reply:
Oh, gotcha. Thanks for clarifying.